### ISSUE TITLE
"no extension found for this protocol" messages printed

### ISSUE THREAD

**Issue by majek** [ldx/python-iptables#137]
What does this mean:

```
/lib/xtables/libxt_set.so: no "set" extension found for this protocol
/lib/xtables/libxt_comment.so: no "comment" extension found for this protocol
/lib/xtables/libxt_standard.so: no "standard" extension found for this protocol
/lib/xtables/libxt_set.so: no "set" extension found for this protocol
/lib/xtables/libxt_comment.so: no "comment" extension found for this protocol
/lib/xtables/libxt_standard.so: no "standard" extension found for this protocol
/lib/xtables/libxt_set.so: no "set" extension found for this protocol
/lib/xtables/libxt_comment.so: no "comment" extension found for this protocol
/lib/xtables/libxt_standard.so: no "standard" extension found for this protocol
```

**Comment by majek** [comment]
Looks like it happens when calling `list(chain.rules)`. Weird.

**Comment by ldx** [comment]
Do you have code that reproduces this?

**Comment by majek** [comment]
I'm really puzzled myself. I can't even find this string "no extension found for this protocol" in any of the sources.

**Comment by majek** [comment]
Here is the bastard:

```
/lib$ strings libxtables.so.10.0.0|grep "exte"
%s: no "%s" extension found for this protocol
```

**Comment by ldx** [comment]
This might happen if you use both IPv4 and IPv6. Latest version of iptables does not print this error message.

Let me know if this causes any issues besides the message on stderr.

**Comment by majek** [comment]
Ok. BTW, this looks simillar https://github.com/ldx/python-iptables/issues/29

**Comment by ldx** [comment]
Yep, even the test suite produces this message with certain versions of iptables (when python-iptables is trying to load certain extensions for IPv6 that had already been loaded for IPv4). Ugly, but as far as I see there are no other side effects.

**Comment by majek** [comment]
Confirmed. This is iptables being shitty. All works okay.

BTW, it would be nice if there was a well docmented and reliable way of pointing python-iptables to proper modules. I'm aware of XTABLES_LIBDIR, but then libxtables.so is being picked up from /lib. Then I set LD_LIBRARY_PATH but then libipset.so is being taken wrong, etc. I have not yet found a reliable way of running python-iptables with custom build. 

    LD_LIBRARY_PATH=.. XTABLES_LIBDIR=.. python x.py
